src/main/java/ch/njol/skript/structures/StructUsing.java

```diff
@@ -0,0 +1,92 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.structures;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
+import org.skriptlang.skript.lang.entry.EntryContainer;
+import org.skriptlang.skript.lang.experiment.Experiment;
+import org.skriptlang.skript.lang.structure.Structure;
+
+@Name("Using Experimental Feature")
+@Description({
+	"Place at the top of a script file to enable an optional experimental feature.",
+	"For example, this might include "
+})
+@Examples({
+	"using 1.21",
+	"using my-cool-addon-feature"
+})
+@Since("INSERT VERSION")
+public class StructUsing extends Structure {
+
+	public static final Priority PRIORITY = new Priority(15);
+
+	static {
+		Skript.registerSimpleStructure(StructUsing.class, "using <.+>");
+	}
+
+	private @UnknownNullability Experiment experiment;
```

Comment:
same here as before regarding unknown nullability

assuming the syntax contact is followed this should effectively be NotNull

Reply:
I still really don't like this because it is _not_ NotNull but I'll compromise my values for you :(